tests/pre/conftest.py

```diff
@@ -0,0 +1,26 @@
+from typing import Iterable, Optional
+
+import pytest
+from eth_typing import ChecksumAddress
+from nucypher.blockchain.eth.agents import (
+    StakingProvidersReservoir,
+    TACoApplicationAgent,
+)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def mock_sample_reservoir(testerchain, mock_contract_agency):
+    def mock_reservoir(
+        without: Optional[Iterable[ChecksumAddress]] = None, *args, **kwargs
+    ):
+        addresses = {
+            address: 1
+            for address in testerchain.stake_providers_accounts
+            if address not in without
+        }
+        return StakingProvidersReservoir(addresses)
+
+    # TODO - this is needed for PRE Policy.enact(...) sample functionality which
+    #  uses TACoApplication - should we change this (in `nucypher`)?
```

Comment:
Yes, but it has low priority for the moment. Opened https://github.com/nucypher/nucypher/issues/3346

Reply:
Thanks for filing 🙏